src/main/java/exception/InvalidMoneyAmountException.java

```diff
@@ -0,0 +1,3 @@
+package exception;
+
+public class InvalidMoneyAmountException extends Exception { }
```

Comment:
Provide detailed exception details / error messages such as "amount to small, not enough balance, ..." @dave7m